content/acra/configuring-maintaining/general-configuration/acra-rollback.md

```diff
@@ -77,6 +77,212 @@ Zones are deprecated since 0.94.0, will be removed in 0.95.0.
   File for store inserts queries. 
   Default is `decrypted.sql`.
 
+#### TLS
+
+* `--tls_database_enabled=<true|false>`
+
+  Turns on/off TLS for connection with Database to `--db_connection_string`.
+
+  * `true` — turns on
+  * `false` — (default) turns off.
+
+* `--tls_auth=<mode>`
+
+  Set authentication mode that will be used for TLS connection.
+
+  * `-1` — not specified, common `--tls_ca` value will be used.
```

Comment:
base `tls_ca` doesn't use common `tls_ca` value. and found just now that description about `common tls_ca` flag is incorrect for `tls_*_auth` flags, because they will use common `tls_auth` flag instead...